mlflow_tracking_server/files/run.sh

```diff
@@ -12,18 +12,8 @@ if [ -z $AWS_BUCKET ]; then
   exit 1
 fi
 
-if [ -z $AWS_ACCESS_KEY_ID ]; then
-  echo >&2 "AWS_ACCESS_KEY_ID must be set"
-  exit 1
-fi
-
-if [ -z $AWS_SECRET_ACCESS_KEY ]; then
-  echo >&2 "AWS_SECRET_ACCESS_KEY must be set"
-  exit 1
-fi
-
 mkdir -p $FILE_DIR && mlflow server \
-    --backend-store-uri file://${FILE_DIR} \
+    --backend-store-uri sqlite:///${FILE_DIR}/sqlite.db \
     --default-artifact-root s3://${AWS_BUCKET}/artifacts \
```

Comment:
Thank you very much for your work. I have only one small comment:
We need the relevant AWS envs (`AWS_BUCKET, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY`) for our S3 bucket access. Could you please undo the deletion of the AWS envs?

Reply:
@flmu I don't believe that I deleted the environment variables; you can still pass them in the same way as described in the README (actually, I'm passing them in that way as well). The only thing was that I deleted the undocumented defaults for the environment variables, which aren't valid anyway.
```dockerfile
ENV AWS_BUCKET bucket
ENV AWS_ACCESS_KEY_ID aws_id
ENV AWS_SECRET_ACCESS_KEY aws_key
```
Is this okay?